src/ngSanitize/sanitize.js

```diff
@@ -218,6 +219,11 @@ function $SanitizeProvider() {
   htmlParser = htmlParserImpl;
   htmlSanitizeWriter = htmlSanitizeWriterImpl;
 
+  nodeContains = window.Node.prototype.contains || /** @this */ function(arg) {
+    // eslint-disable-next-line no-bitwise
+    return !!(this.compareDocumentPosition(arg) & 16);
```

Comment:
`compareDocumentPosition` can also be clobbered. That will throw an exception - is that OK for your case i.e. is the exception caught and safely resolved?